spec/suites/map/MapSpec.js

```diff
@@ -2206,6 +2206,31 @@ describe("Map", () => {
 		});
 	});
 
+	describe("#project", () => {
+
+		it("throws if map is not set before", () => {
+			expect(() => {
+				map.panInsideBounds();
+			}).to.throwError();
+		});
+
+		it("returns geographical coordinate relative to the CRS origin", () => {
```

Comment:
The code is good, but the title is (slightly) misleading. `map.project()` returns *pixel coordinates* relative to the top-left of the CRS extents; it does **not** return geographical coordinates of any kind (in any valid CRS).

I must ask you to change the title (and look for a typo in the other test as well). It'd be nice to add another test with the same `LatLng` but zoom level `6`, to see that the return value indeed doubles.